sky/utils/cli_utils/status_utils.py

```diff
@@ -325,6 +333,14 @@ def _is_pending_autostop(cluster_record: _ClusterRecord) -> bool:
 # ---- 'sky cost-report' helper functions below ----
 
 
+def _get_status__value_for_cost_report(
+        cluster_cost_report_record: _ClusterCostReportRecord) -> int:
+    status = cluster_cost_report_record['status']
+    if status is None:
+        return -1
+    return 1
```

Comment:
Why are we sorting a bunch of -1's and 1's? From the output, seems like we are sorting by launch-time?

Reply:
We wanted to bubble up clusters that were active and un-terminated to the top since they are actively incurring cost. In addition, we also sort by launch time.

Reply:
Can we put this comment as comment before L172?